lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -57,6 +57,20 @@ inline Dst bitwiseCast(Src source) noexcept {
   return dest;
 }
 
+static bool extractValueFromConstOp(Operation *op, int32_t &indexValue) {
```

Comment:
This should return LogicalResult instead of bool

Reply:
Thanks, fixed.